### PR TITLE
Plug-in 3.0 and vRA 7.3 updates

### PR DESCRIPTION
**Pull request by jpadams**
This update drops the legacy blueprints in favor of pointing to docs on using the new Puppet component to create blueprints. The old content is preserved on a branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
-# vRealize Automation Plug-in Starter Content Pack
+# vRealize Automation Plug-in Starter Content Pack for vRA 7.3 and Puppet plug-in 3.0
```
Comment (turbodog) [inline README.md]: Puppet plug-in for vRealize

File: README.md [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
 
-Please see the [complete documentation](https://docs.puppet.com/pe/latest/vro_intro.html) for more information on how to install the Puppet plug-in and use the content provided in this repository.
+## Getting started
+
+Here's your high-level plan to get started:
+- Create a Puppet Enterprise master and follow the instructions at the bottom of this page to install the starter content.
+- Remove any previous Puppet plug-ins and install the [Puppet plug-in for vRealize Automation 3.0](https://solutionexchange.vmware.com/store/products/puppet-plugin-for-vrealize-automation) into the embedded vRO that comes in your vRA appliance (external vRO servers are not supported, use the legacy workflow if you need this).
```
Comment (turbodog) [inline README.md]: "(external vRO servers are not supported, use the legacy workflow if you need this)" --> "External vRO servers are not supported in vRA 7.3. If you need to use an external vRO server, then use the version 2.0 plug-in"

@jpadams what if you only have vRO? does the statement above mean that v3.0 isn't expected to work?

File: README.md [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
+- Follow the vRA 7.3 docs to add a Puppet endpoint (points to your master), and create your Puppet Enterprise blueprints using code from this repo.
```
Comment (turbodog) [inline README.md]: Add the link in "the vRA 7.3 docs"

File: README.md [hunk 4]
```diff
@@ -1,19 +1,34 @@
...
+
+> Note: if you're looking for the 6.x, 7.x (pre 7.3) content including example blueprints, you can still find that in this repo on the "iaas_eventbroker" branch.
```
Comment (turbodog) [inline README.md]: " Note: if you're looking for the 6.x, 7.x (pre 7.3) content including example blueprints, you can still find that in this repo on the "iaas_eventbroker" branch." --> "Note: if you are using vRealize 6.x or 7.0 through 7.2, then use the "iaas_eventbroker" branch of this repo."

File: README.md [hunk 5]
```diff
@@ -1,19 +1,34 @@
...
+
+vRA 7.3 Puppet docs:
+- [Create a Puppet Endpoint](http://pubs.vmware.com/vrealize-automation-73/index.jsp#com.vmware.vra.prepare.use.doc/GUID-7F7059C8-E80F-42E8-B0AE-32F794C6FC38.html)
+- [Creating Puppet Enabled vSphere Blueprints](http://pubs.vmware.com/vrealize-automation-73/index.jsp#com.vmware.vra.prepare.use.doc/GUID-45BF018B-0C25-489D-89AA-8A7C91E7E9A6.html)
+- [Add a Puppet Component to a vSphere Blueprint](http://pubs.vmware.com/vrealize-automation-73/index.jsp#com.vmware.vra.prepare.use.doc/GUID-BB99F78C-1638-4852-92B7-30348E8EBBA2.html)
+- [vRealize Automation 7.3 Release Notes](http://pubs.vmware.com/Release_Notes/en/vra/73/vrealize-automation-73-release-notes.html)
+
+Please see the [complete documentation](https://docs.puppet.com/pe/latest/vro_intro.html) for more detailed information.
```
Comment (turbodog) [inline README.md]: "complete documentation" --> "Puppet plug-in for vRealize documentation"

**Review by turbodog — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jpadams) [reply README.md]: I need to word this better. I believe that only the interval vRO is supported with the vRA/Puppet GUI integration. External vRO's will still work with Event Broker.

File: README.md (hunk 4 above)
Comment (jpadams) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (jpadams) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (jpadams) [reply README.md]: done. linked to Puppet endpoint docs.

File: README.md (hunk 2 above)
Comment (jpadams) [reply README.md]: fixed up.

File: README.md (hunk 1 above)
Comment (jpadams) [reply README.md]: done. left 3.0 on there as well.

**Comment by turbodog** [comment]
LGTM. I don't have merge rights though.

**Comment by jpadams** [comment]
I don't stand on ceremony.

**Pull request merged**